account_advanced_reconcile_compassion/wizard/reconcile_fund_wizard.py

```diff
@@ -125,12 +132,12 @@ def reconcile_with_fund(self, cr, uid, ids, context=None):
         return {'type': 'ir.actions.act_window_close'}
 
     def _generate_invoice_line(self, cr, uid, invoice_id, product, price,
-                               partner_id, context=None):
+                               partner_id, contract_ids, context=None):
 
         inv_line_data = {
             'name': product.name,
             'account_id': product.property_account_income.id,
-            'price_unit': price,
+            'price_unit': price / len(contract_ids),

```

Comment:
As you divide the price by nb of contract to get the unit price, shouldn't the quantity be the nb of contract to get the price of the invoice line be the actual price given to the function ?